resolwe_bio/processes/workflows/rnaseq.yml

```diff
@@ -229,3 +229,103 @@
           mode: union
           stranded: '{{input.stranded}}'
           id_attribute: '{{input.id_attribute}}'
+
+- slug: workflow-meta-rnaseq
+  name: RNA-Seq workflow (meta)
+  requirements:
+    expression-engine: jinja
+  data_name: "Pipeline ({{ reads|sample_name|default('?') }})"
+  version: 0.0.1
+  type: data:workflow:rnaseq:htseq
+  category: workflow:primary
+  persistence: RAW
+  description: >
+    Run RNA-Seq workflow
+  input:
```

Comment:
This guys should have simple inputs, actually the same as the descriptor schema. The inputs should make sense to biologists.